### PR TITLE
Update to 1.3.6

### PR DESCRIPTION
**Pull request by JeanChristopheMorinPerso**
Upstream: https://github.com/openai/openai-python/tree/v1.3.6

Channel: defaults

Depends on:
* https://github.com/AnacondaRecipes/distro-feedstock/pull/1

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,50 +1,45 @@
...
-    - colorama
-    - typing_extensions  # [py<38]
+    - anyio <4,>=3.5.0
+    - distro <2,>=1.7.0
+    - httpx <1,>=0.23.0
+    - pydantic <3,>=1.9.0
+    - sniffio
+    - tqdm >4
+    - typing-extensions <5,>=4.5
     # The following are required for the optional things.
```
Comment (lorepirri) [inline recipe/meta.yaml:34]: if these are optional, shouldn't they go to `run_constrained`?
https://github.com/openai/openai-python/blob/17ac6779958b2b74999c634c4ea4c7b74906027a/pyproject.toml#L39

**Review by lorepirri — APPROVED** [review]

File: recipe/meta.yaml (hunk 1 above)
Comment (JeanChristopheMorinPerso) [reply recipe/meta.yaml:34]: It would technically be a breaking change to our users... We can but we need to consider the impact.

Comment (danpetry) [reply recipe/meta.yaml:34]: numpy is used to return [embedding vectors](https://en.wikipedia.org/wiki/Word_embedding) in base64 format: https://github.com/openai/openai-python/blob/17ac6779958b2b74999c634c4ea4c7b74906027a/src/openai/resources/embeddings.py#L85

pandas is used to validate data (before sending it to the API maybe?) https://github.com/openai/openai-python/blob/17ac6779958b2b74999c634c4ea4c7b74906027a/src/openai/lib/_validators.py#L10

this package isn't used as a dependency much. Maybe putting these as run_constrained would be better to improve co-installability? I don't think it matters a great deal either way though.

**Review by danpetry — APPROVED** [review]

File: recipe/meta.yaml (hunk 1 above)
Comment (danpetry) [reply recipe/meta.yaml:34]: I think this is pretty marginal either way

Comment (JeanChristopheMorinPerso) [reply recipe/meta.yaml:34]: I think I'll leave them there for now.

**Pull request merged**